packages/mobile/views/dashboard/drawers/account-actions/views/AccountActionsView.svelte

```diff
@@ -17,4 +40,25 @@
             {localize('actions.customizeAcount')}
         </Button>
     {/if}
+    {#if features?.dashboard?.accountActions?.toggleVisibility?.enabled}
+        <Button
+            outline
+            disabled={!$selectedAccount.hidden && $nonHiddenActiveAccounts.length <= 1}
+            onClick={() => ($selectedAccount.hidden ? handleShowAccountClick() : handleHideAccountClick())}
+            icon={$selectedAccount.hidden ? Icon.View : Icon.Hide}
+        >
+            {localize($selectedAccount.hidden ? 'actions.showAccount' : 'actions.hideAccount')}
+        </Button>
+    {/if}
+    {#if features?.dashboard?.accountActions?.toggleVisibility?.enabled}
+        <HR />
+        <Button
+            variant={ButtonVariant.Warning}
+            icon={Icon.Delete}
+            disabled={!$selectedAccount.hidden && $nonHiddenActiveAccounts.length <= 1}
```

Comment:
The message isn't right?
![image](https://user-images.githubusercontent.com/6748220/215429768-88c1a9b6-4a95-415d-b340-ba444547f9b2.png)

The delete button for the same account is hidden on desktop:
![image](https://user-images.githubusercontent.com/6748220/215429706-c807f564-b13d-4fa3-ba8e-05d7681cca03.png)

Reply:
The message isnt right, opened discussion with the team, the issue is also found on desktop